src/SyntacticModelsBase.jl

```diff
@@ -23,6 +24,7 @@ _dict(x::Symbol) = x
 _dict(x::String) = x
 _dict(x::Number) = x
 _dict(x::AbstractVector) = map(_dict, x)
+_dict(x::DiagrammaticEquations.decapodes.Judgement) = x
```

Comment:
@jpfairbanks This line feels suspect to me, but is what got the JSON to generate and tests to pass.